src/thread.rs

```diff
@@ -293,10 +299,10 @@ impl<'a> Scope<'a> {
     /// scope exits.
     ///
     /// [spawn]: http://doc.rust-lang.org/std/thread/fn.spawn.html
-    pub fn spawn<'s, F, T>(&'s self, f: F) -> ScopedJoinHandle<'a, T>
+    pub fn spawn<F, T>(&self, f: F) -> ScopedJoinHandle<'a, T>
     where
-        'a: 's,
```

Comment:
`'a: `s` is unnecessary because `'a` is at least as big as the scope, while `'s` is smaller than the scope.